### DISCUSSION TITLE
Second bluetooth speaker doesn`t work

### DISCUSSION THREAD

**Discussion by jialarcon** [pschatzmann/ESP32-A2DP#223] [Q&A] [answered]
Hi, I have succesfully played a recorded audio in a bluetooth speaker using the library. The problem is that if I try to use a different speaker it doesn't work, it will always play in the first speaker, even if I change the name in "a2dp_source.start".
Any clues ?
Kind Regards

Jose Ignacio

**Comment by pschatzmann** [comment, marked as answer]
https://github.com/pschatzmann/ESP32-A2DP/wiki/Auto-Reconnect

**Reply by jialarcon** [reply]
Thank you, it worked after adding in my setup section

a2dp_source.set_auto_reconnect(false);

Kind Regards

Jose Ignacio
